docs/build/isc/v1.0.0-rc.6/docs/how-tos/token/erc20-native-token.md

```diff
@@ -8,25 +8,30 @@ tags:
   - how-to
 ---
 
-# Custom ERC20 functions
+# Custom ERC20 Functions
 
-Once you [registered your native token as ERC20](./erc20-native-token.md) you can basically use it like any other ERC20 token, with functions like `transfer`, `balanceOf` and so on. But as the ERC20 token maps the native token on L2 there are some additional fetures.
+Once you [registered your native token as ERC20](./erc20-native-token.md) you can use it like any other ERC20 token, with functions like
+`transfer`, `balanceOf`, etc. But, as the ERC20 token maps the native token on L2, there are some additional ISC features 
+you can take advantage of.
 
 ## Example Code
 
-1. Get your `ERC20NativeTokens` instance (In this case we will use `erc20NativeTokensAddress` and the Foundry serial number to get the contract address)
+### Get Your `ERC20NativeTokens` Instance
```

Comment:
I wouldn‘t add another title here. Especially as this is more about the nativeTokenID function of an ERC20NativeTokens object